__tests__/bundle/bundle-lint-format/checkstyle-format-snapshot.js

```diff
@@ -9,6 +9,7 @@ exports[`E2E bundle lint format bundle lint: should be formatted by format: chec
 </checkstyle>
 
 No configurations were defined in extends -- using built in recommended configuration by default.
+Warning! This default behavior is going to be deprecated soon..
```

Comment:
Let's have only one snapshot with this warning and keep other tests with happy-path.